src/client/pages/Offer/ProductSelector/Selector.tsx

```diff
@@ -61,58 +61,73 @@ export const Selector = ({
     })
   }
 
+  const areChoicesAvailable =
+    standaloneProducts.length + additionalProducts.length > 1
+  if (!areChoicesAvailable) {
+    return null
+  }
+
   return (
     <Wrapper className={className}>
```

Comment:
Nit: JSX for this component is huge.  Can we extract main and additional sections into separate components without too much overhead